python/paddle/v2/fluid/tests/book_distribute/test_dist_image_classification.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved
```

Comment:
Can you please split the fix and the book dist sample in two PRs?

Reply:
Done.